Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:11-alpine
+
+COPY . .
```

Comment:
When we move this down we will more often be able to use a cached intermediate container.